src/gui/screen_menu_esp_update.cpp

```diff
@@ -160,7 +160,7 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
             esp_loader_connect_args_t config = ESP_LOADER_CONNECT_DEFAULT();
             _dbg("ESP boot connect");
             if (ESP_LOADER_SUCCESS == esp_loader_connect(&config)) {
-                help.SetText(_("Successfully connected to ESP module and switched into UART bootloader."));
+                help.SetText(_("Successfully connected to ESP, do not swith the Printer off nor remove the Flash disk."));
```

Comment:
Minor suggestion: Split into two sentences? `Successfully connected to ESP. Do not switch…`?

Reply:
ok